health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/index.js

```
@@ -201,7 +203,7 @@ const App = ({ path, stateCode, userType, tenants, BOUNDARY_HIERARCHY_TYPE, hier
         </React.Fragment>
         <PrivateRoute path={`${path}/setup-microplan`} component={() => <SetupMicroplan hierarchyType={BOUNDARY_HIERARCHY_TYPE} hierarchyData={hierarchyData} />} />
         <PrivateRoute path={`${path}/microplan-search`} component={() => <MicroplanSearch></MicroplanSearch>} />
-        <PrivateRoute path={`${path}/user-management`} component={() => <UserManagement></UserManagement>} />
+        <PrivateRoute path={`${path}/user-management`} component={() => <UserManagement ></UserManagement>} />
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)